### ISSUE TITLE
Support more package managers

### ISSUE THREAD

**Issue by kballard** [yonaskolb/Mint#67]
I'm glad to see you support one package manager, but I don't use Homebrew anymore since the lead maintainer is a huge jackass.

Would you consider supporting at least one other package manager? The obvious choice is [MacPorts](https://www.macports.org). Personally I'm also a fan of [Nix](https://nixos.org/nix/), though I realize that one's a bit esoteric.

**Comment by yonaskolb** [comment]
Yeah, I'm not the biggest fan of Homebrew either. At least when Mint is installed it can update itself 😄 
I'd suggest using SPM itself to install otherwise
```
swift run mint run yonaskolb/mint
```

If you'd like to submit Mint to any package manager or create a PR here to add support, feel free. I'm happy to have multiple install methods, as long as the release step is scriptable

**Comment by kballard** [comment]
I'm not aware of any equivalent to `brew bump-formula-pr` for other package managers, so if you're going to require that it be scriptable, then I'm not sure how to handle that.

**Comment by kballard** [comment]
Here's a thought: Maybe provide a one-line shell script, something like `curl https://raw.githubusercontent.com/yonaskolb/Mint/master/install.sh | sh`, that clones the repo to a temporary directory and runs `swift run mint install yonaskolb/mint` inside it?
